Week2_LibraryTask/hardhat.config.ts

```diff
@@ -0,0 +1,42 @@
+require("dotenv").config();
+import { HardhatUserConfig, task, subtask } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+
+const config: HardhatUserConfig = {
+  solidity: "0.8.18",
+  networks: {
```

Comment:
If you don't want to specify network every time using the cli `--network` you can add a `defaultNetwork` field in the config. https://hardhat.org/hardhat-runner/docs/config

Reply:
That's pretty cool, thanks for the tip!